### PR TITLE
BD/SP Random Battle: Update movepools

### PR DESCRIPTION
**Pull request by Tarrembeau**

### CODE REVIEW

File: data/mods/gen8bdsp/formats-data.ts [hunk 1]
```diff
@@ -227,7 +227,7 @@ export const FormatsData: {[k: string]: ModdedSpeciesFormatsData} = {
 	persian: {
 		tier: "RU",
 		doublesTier: "DUU",
-		randomBattleMoves: ["fakeout", "slash", "uturn", "bite", "aerialace"],
+		randomBattleMoves: ["fakeout", "covet", "uturn", "bite", "aerialace"],
```
Comment (AnnikaCodes) [inline data/mods/gen8bdsp/formats-data.ts:230]: Is this really better? Is Persian without an item so often that it's worth giving up the extra damage from Slash?

Comment (Karthik99999) [reply data/mods/gen8bdsp/formats-data.ts:230]: Covet gets boosted by Technician

Comment (AnnikaCodes) [reply data/mods/gen8bdsp/formats-data.ts:230]: Oh, right, I forgot about Technician. Sure, that's an improvement then.

**Review by AnnikaCodes — APPROVED** [review]
Looks good, thanks!

**Pull request merged**